prettypandas/styler.py

```diff
@@ -292,11 +313,15 @@ def as_money(self,
                                       currency=currency,
                                       precision=precision,
                                       subset=subset,
-                                      location=location)
+                                      location=location, 
+                                      **kwargs)
 
     def _format_cells(self, func, subset=None, **kwargs):
         """Add formatting function to cells."""
 
+        if self.replace_all_nans_with is not None and 'replace_nan_with' not in kwargs:
+            kwargs = dict(replace_nan_with = self.replace_all_nans_with, **kwargs)
```

Comment:
Any reason not to use `kwargs.update(replace_nan_with=self.replace_all_nans_with)`?